bedevere/bpo.py

```diff
@@ -37,6 +51,14 @@
         else:
             status = FAILURE_STATUS
     else:
+        if "body" in event.data["pull_request"]:
+            body = event.data["pull_request"]["body"]
+            bpo_url_found = BPO_URL_RE.search(body)
```

Comment:
This is something similar I did at first (personally I don't use re module that much). I will use this msg text now.